Header header
uint8 color_index
string color_name